tests/commit/merge.c

```diff
@@ -0,0 +1,66 @@
+#include "clar_libgit2.h"
```

Comment:
I hate to be nitpicky, but I would have expected the test to be part of `tests/refs/reflog/reflog.c`. Sorry to not state this earlier